libs/plugins/stream-client/src/lib/params-schema/param-row-input/param-row-input.component.less

```diff
@@ -26,34 +33,37 @@
 .form-input__delete {
   justify-content: center;
   align-items: center;
-  width: 45px;
-  height: 45px;
+  width: 40px;
+  height: 100%;
   font-weight: bold;
   border: 0;
   display: flex;
-  margin-left: 5px;
+  margin-left: 16px;
 
   &:hover .delete-icon {
-    color: @color-secondary;
+    color: @color-primary;
   }
 }
 
 .type-selector {
-  margin-right: 20px;
-  margin-left: 20px;
+  flex: 1;
+  margin-right: 24px;
+  margin-left: 24px;
 }
 
 .type-selector__label {
-  width: 100px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
   text-align: left;
-  height: 45px;
+  height: 100%;
   position: relative;
 }
 
 .type-selector__caret {
-  position: absolute;
-  top: 22px;
-  right: 22px;
+  color: #79b8dc;
```

Comment:
As we are following the practice of maintaining a variable for reusable properties, use a variable here.

Reply:
sure Praneeth